esrally/driver/runner.py

```diff
@@ -52,7 +52,11 @@ def register_default_runners():
     register_runner(track.OperationType.RawRequest, RawRequest(), async_runner=True)
     register_runner(track.OperationType.Composite, Composite(), async_runner=True)
     register_runner(track.OperationType.SubmitAsyncSearch, SubmitAsyncSearch(), async_runner=True)
-    register_runner(track.OperationType.GetAsyncSearch, Retry(GetAsyncSearch(), retry_until_success=True), async_runner=True)
+    register_runner(
```

Comment:
Thank you for the careful review! I tried a few line length settings, but didn't realize they were "sticky". I fixed this by rebasing and applied `make format` on the original code, which fixed it, as you mentioned.

(It took me some time because I had to work around https://github.com/psf/black/issues/2321.)